gdpr/vendorlist-fetching.go

```diff
@@ -158,7 +158,7 @@ func newVendorListCache(fallbackVL api.VendorList) (save func(id uint16, list ap
 		if ok {
 			return list.(vendorlist.VendorList)
 		}
-		return fallbackVL
+		return nil
```

Comment:
This method needs to return `nil` in order to attempt a url lazy loading fetch. If we return `fallbackVL` here, it effectively disables lazy loading.